server/v2/testdata/app.toml

```diff
@@ -19,10 +25,10 @@ minimum-gas-prices = '0stake'
 app-db-backend = 'goleveldb'
 
 [store.options]
-# State storage database type. Currently we support: 0 for SQLite, 1 for Pebble
-ss-type = 0
-# State commitment database type. Currently we support:0 for iavl, 1 for iavl v2
-sc-type = 0
+# SState storage database type. Currently we support: "sqlite" and "pebble"
```

Comment:
```suggestion
# SState storage database type. Currently we support: "sqlite", "pebble" and "rocksdb"
```

Reply:
Do we really want to add it?
It won't work it seems
```
		// TODO: rocksdb requires build tags so is not supported here by default
		return nil, errors.New("rocksdb not supported")
```

Reply:
dont think this is a todo, its just that it requires build flags. Its suppose be supported out of the box

Reply:
made the needed changes here https://github.com/cosmos/cosmos-sdk/pull/21649